UsingSeal.md

```diff
@@ -254,49 +262,147 @@ Start by generating a master seed for the key server.
 Use the `seal-cli` tool as `cargo run --bin seal-cli gen-seed`.
 This command outputs the secret master seed which should be stored securely.
 
+```shell
+cargo run --bin seal-cli gen-seed
+Seed: $MASTER_SEED
```

Comment:
$MASTER_SEED -> example output?

Reply:
yeah its a value that comes from this output, so its easy to understand where it comes from. its not an environment variable, i should use <MASTER_SEED> instead.